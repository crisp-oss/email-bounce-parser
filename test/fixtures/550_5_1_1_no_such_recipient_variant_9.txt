This is the mail system at host mailer.acme.email.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                   The mail system

<thomas@dynamiciptv.com>: host route2.mx.cloudflare.net[126.195.250.81] said:
    550 5.1.1 Address does not exist. cEM8XVXAY905 (in reply to RCPT TO
    command)
