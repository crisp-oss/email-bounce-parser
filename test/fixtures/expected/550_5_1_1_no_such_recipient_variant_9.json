{
  "bounce": true,
  "email": {
    "body": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system\n\n<thomas@dynamiciptv.com>: host route2.mx.cloudflare.net[126.195.250.81] said: 550 5.1.1 Address does not exist. cEM8XVXAY905 (in reply to RCPT TO command)",
    "intro": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system",
    "error": "<thomas@dynamiciptv.com>: host route2.mx.cloudflare.net[126.195.250.81] said: 550 5.1.1 Address does not exist. cEM8XVXAY905 (in reply to RCPT TO command)"
  },
  "data": {
    "error": {
      "code": {
        "basic": "550",
        "enhanced": "5.1.1"
      },
      "label": "Address does not exist. cEM8XVXAY905",
      "type": "action_not_taken",
      "temporary": false,
      "permanent": true,
      "data": {
        "type": "no_such_recipient"
      }
    },
    "recipient": "thomas@dynamiciptv.com",
    "server": {
      "hostname": "route2.mx.cloudflare.net",
      "ip": "126.195.250.81",
      "port": null
    },
    "command": "RCPT TO"
  }
}